### ISSUE TITLE
workbench.action.terminal.focus is disappear after an update

### ISSUE THREAD

**Issue by inusedname** [microsoft/vscode#128639]
After update to July 2021 Update (1.58.1), I realized that the cursor does not move automatically to terminal after I run code from "Code Runner" extension. I rechecked the setting and found that workbench.action.terminal.focus has been faded off and show "Unknown Configuration Setting". Is there a way to enable it again, or another way to change focus to terminal ?
![image](https://user-images.githubusercontent.com/49682088/125611672-474d8043-307e-49ea-abd3-4b32070cded2.png)

**Comment by denishowew** [comment]
I have the same problem when opening my first new integrated terminal (e.g. with Ctrl-`).  The terminal no longer gets input focus.  Terminals added after the first _do_ get focus.

The same happens with a fresh ~/Library/Application Support/Code.

Version: 1.58.1
Commit: 2aeda6b18e13c4f4f9edf6667158a6b8d408874b
Date: 2021-07-13T06:22:33.013Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Darwin x64 19.6.0 (macOS 10.15.7)

**Comment by husjon** [comment]
#128255

**Comment by joaomnb** [comment]
Yep, 1.58.2 fixed it for me as well.

```
Version: 1.58.2
Commit: c3f126316369cd610563c75b1b1725e0679adfb3
Date: 2021-07-14T22:09:06.581Z (1 day ago)
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Darwin x64 20.4.0
```

**Comment by inusedname** [comment]
Problem solved with 1.58.2 patch.
